en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -1,96 +1,96 @@
+    ![View latency details](../assets/img/observability/latency-details.png)
+    
+- **Viewing details at a higher granularity**
    
-    !!!tip
-        You can expand the graph by clicking and dragging the cursor over the period you want to drill down.
-        
-2. Observe the latencies of each request listed in the low-code diagram.
-3. Click on one request shown on the low-code diagram (latencies). You can observe the status code for that request and thereby identify the exact API invocation that caused any error(s).
+    If you want to drill down to view details at a higher granularity, drag the pointer over a time range to view the graph for the selected period.
+    
+- **Checking the status code of a request that has caused a latency**
 
-### Step 2: Drill into the analytics
+    To do this, click on the relevant latency value. In this example, the status code is `500`, which means the request resulted in an error state.
 
-![Diagnostic view](../assets/img/observability/diagnostics-view.png){.cInlineImage-full}
 
-Follow the steps below to learn how to drill down and confirm the root cause for the anomalies we detected in the **Latency and Throughput** graphs.
+### Analyze diagnostic data
 
-1. Click on the **Diagnostics View** tab on the left panel. 
+The **Diagnostic View** allows you to further drill down the observability data you viewed in the **Throughput & Latency** tab to identify possible root causes for anomalies of a service.
 
-    !!!info
-        The time range selected for the **Throughput & Latency** graphs is applied to the **Diagnostics View** by default. We recommend capturing the time range you want to analyze in detail from the **Throughput & Latency** graph and then navigate to the **Diagnostics View**.
+The following image is an example of how you can view diagnostic data once you click on the **Diagnostics View** tab.
+ 
+![Diagnostic data](../assets/img/observability/diagnostics-view.png){.cInlineImage-full}
+
+!!!info
+    The time range applied to the **Diagnostics View** by default is the same time range for which you viewed the **Throughput & Latency** before clicking on this tab. 
         
-2. In the sample data, you can observe a couple of prominent errors appearing in the four bins.
-3. Select a bin that shows a higher occurrence of the error log "error while connecting to the hr-service”. Notice that this bin may include other errors which occurred at a lesser frequency.
+If you hold the pointer over a section where you see fluctuations in multiple bins, you can view the related logs.
+
+![Fluctuations in diagnostic data](../assets/img/observability/second-bin.png){.cInlineImage-full}
 
-    ![Diagnostic view - second bin](../assets/img/observability/second-bin.png){.cInlineImage-bordered}
+In this example, the logs indicate a higher occurrence of `error while connecting to the hr-service`. Although there are other errors also displayed, those error counts are comparatively much lower.
```

Comment:
Shouldn't we say how to conclude that we have a higher occurrence of this error?